### PR TITLE
Update app.py

### PR DESCRIPTION
**Pull request by sejalapeno**
I did not change find_one, insert_one, update_one, Student_Request and log_action as I was unsure if those are reserved keywords. Let me know if any change is needed.

### CODE REVIEW

**Comment by jibranabsarulislam** [comment]
hey @sejalapeno, thanks for the PR!

https://github.com/It-s-Saturday/Fornear/blob/bba7eae4373562b7ee42ff88e907e6915bf2b8e9/frontend/src/components/staff/RequestTable.jsx#L28-L34

https://github.com/It-s-Saturday/Fornear/blob/bba7eae4373562b7ee42ff88e907e6915bf2b8e9/backend/app.py#L196-L204

this change should affect data fields only. typically in Python the convention is to use snake case, so the functions and python variables can remain unchanged.

The two snippets above show where the data should be named `dateFulfilled` instead of `date_fulfilled`, requiring a change both in frontend files whose data fields currently use snake case, and backend files whose data fields (not functions/variables) also use snake case.

**Comment by sejalapeno** [comment]
Hey I changed the app.py for backend

**Comment by jibranabsarulislam** [comment]
looks great, thanks for the update @sejalapeno. I think the only usage of camel case in a data field in the frontend is in

https://github.com/It-s-Saturday/Fornear/blob/bba7eae4373562b7ee42ff88e907e6915bf2b8e9/frontend/src/components/staff/RequestTable.jsx#L28-L34

it'd probably be something like removing the underscore and updating `${type}` to the first letter of `type` uppercase

**Comment by sejalapeno** [comment]
However I am not sure for which files in frontend do you want the changes

**Comment by sejalapeno** [comment]
Wait , so only this one file in frontend is to be updated?
Also I am afraid I did not understand the change you want exactly

**Comment by jibranabsarulislam** [comment]
> Wait , so only this one file in frontend is to be updated? Also I am afraid I did not understand the change you want exactly

it should be. the frontend should access data fields that are in camel case. The snippet I linked currently looks at `data_${type}` which should instead by `data${type}`, where type's first character is made uppercase (that uppercase logic can go directly inside the `{...}`

File: frontend/src/components/staff/RequestTable.jsx [hunk 1]
```diff
@@ -28,8 +28,8 @@ export default function RequestTable({ type, refresh, onRefresh }) {
   if (type !== 'unfulfilled') {
     columns.push({
       title: `${headerText} Date`,
-      dataIndex: `date_${type}`,
-      key: `date_${type}`,
+      dataIndex: `date${type=type[0].toUpperCase() + type.slice(1)}`,
```
Comment (jibranabsarulislam) [inline frontend/src/components/staff/RequestTable.jsx:31]:
```suggestion
      dataIndex: `date${type[0].toUpperCase() + type.slice(1)}`,
```

**Review by jibranabsarulislam — CHANGES_REQUESTED** [review]

**Review by jibranabsarulislam — APPROVED** [review]

**Comment by jibranabsarulislam** [comment]
great, thanks a ton!

@all-contributors please add @sejalapeno for code

**Pull request merged**